### ISSUE TITLE
Could not find artifact org.eclipse.jetty.tests:test-webapps-parent:pom:9.4.8.v20171121

### ISSUE THREAD

**Issue by anthonydahanne** [eclipse/jetty.project#2278]
Hello Jetty team,
In a project pom, I have : 
```
    <dependency>
      <groupId>org.eclipse.jetty</groupId>
      <artifactId>jetty-distribution</artifactId>
      <version>${jetty.version}</version>
      <type>tar.gz</type>
      <scope>provided</scope>
    </dependency>
```

with     <jetty.version>9.4.8.v20171121</jetty.version>

Now, when I try to build this project, I have :  
```
Could not resolve dependencies for project com.terracotta:tmc-kit:pom:1.4.5-SNAPSHOT: Failed to collect dependencies at org.eclipse.jetty:jetty-distribution:tar.gz:9.4.8.v20171121 -> org.eclipse.jetty:test-jetty-webapp:war:9.4.8.v20171121: Failed to read artifact descriptor for org.eclipse.jetty:test-jetty-webapp:war:9.4.8.v20171121: Could not find artifact org.eclipse.jetty.tests:test-webapps-parent:pom:9.4.8.v20171121 
```
And if you look at the pom of org.eclipse.jetty:test-jetty-webapp:war:9.4.8.v20171121, it seems to indeed require org.eclipse.jetty.tests:test-webapps-parent:pom:9.4.8.v20171121 : https://search.maven.org/remotecontent?filepath=org/eclipse/jetty/test-jetty-webapp/9.4.8.v20171121/test-jetty-webapp-9.4.8.v20171121.pom , but https://search.maven.org/#search%7Cga%7C1%7Ctest-webapps-parent  does not exist in this version...

**Comment by joakime** [comment]
Yes.  That's not right.
The test artifacts shouldn't be required/needed to use any of the distribution artifacts.

Note, you should probably be using `jetty-home` instead of `jetty-distribution`. (you don't need the `demo-base` and all of its baggage)

``` xml
    <dependency>
      <groupId>org.eclipse.jetty</groupId>
      <artifactId>jetty-home</artifactId>
      <version>${jetty.version}</version>
      <type>tar.gz</type>
      <scope>provided</scope>
    </dependency>
```

**Comment by anthonydahanne** [comment]
okee ! thanks @joakime ! I'm upgrading form 8.1 to 9.4, I used to use distribution, but I get it, home now it is !

**Comment by anthonydahanne** [comment]
hello @joakime  - I found out that home and distribution are 2 different beasts.
If I unzip home, and create a webapps folder and copy my war in it, well using : 
    java -jar start.jar --module=http jetty.http.port=8080
will just start jetty, but that's it.
If I do the same thing with distribution, my war is deployed.
So I guess I'll stick to distribution :-)

**Comment by joakime** [comment]
Read https://www.eclipse.org/jetty/documentation/9.4.x/startup-base-and-home.html

Short (incomplete) summary ...

* The `jetty-home` artifact is the same as the `${jetty.home}` directory. (note: `jetty-distribution` is also a `${jetty.home}`, the below also applies with `jetty-distribution`)
* Do not modify / edit / delete / change / add any content in `${jetty.home}`.  Leave it alone.
* Create `${jetty.base}` directory for your instance specific configuration.
  * Configuration is stored in `${jetty.base}/start.ini` or `${jetty.base}/start.d/*.ini` files
  * All of your webapps, logs, resources, libs, modules, etc are stored in `${jetty.base}`

**Comment by jmcc0nn3ll** [comment]
- jetty-distribution is intended for people to download and deploy their webapps in the traditional manner, where either jetty.home and jetty.base are the same (not recommended) or split out however they choose.  It provides examples and the demo base for people unfamiliar with Jetty to get accustomed to how things work.  Much of our documentation is authored around the usage of jetty as a traditional distribution.

- jetty-home is the basis for building distributions off of, our jetty-distribution being one example but many folks use jetty-home as the basis for docker containers and restful services.  That you are slurping it in as a maven dependency indicates to us you are not looking for a traditional downloadable distro but something that you are customizing in some way, minimally you should be removing the demo-base and things like that.  If you have issues making jetty-home fit your needs then we are all ears on ways to improve it.

**Comment by anthonydahanne** [comment]
hello @joakime @jmcc0nn3ll - thanks for your comment.
You were right, I was not using the jetty home tarball properly - and I found out reading the docs that it's still the right package for me.
Looking at demo-base, I got the necessary inspiration to update our packaging; you were right assuming I am using jetty to ship an application consisting of a war and custom servelt container configuration and launchers.
Thanks for the pointers and the explanation !
